CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## v0.3.0 (Released 2024-06-20)
+
+ADDITIONS
+
+- feat: add primary regulator to business profile
+- examples: add business profile 
+ 
+IMPROVEMENTS
+
+- **breaking**: update CreateTransfer to constrain return params based on async vs sync 
```

Comment:
Note that this does include a minor breaking change on the `CreateTransfer` method